src/Annotation/AccessorOrder.php

```diff
@@ -12,16 +12,37 @@
  *
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
+#[\Attribute(\Attribute::TARGET_CLASS)]
 final class AccessorOrder
 {
     /**
      * @Required
-     * @var string
+     * @var string|null
      */
-    public $order;
+    public $order = null;
 
     /**
      * @var array<string>
      */
     public $custom = [];
+
+    public function __construct(array $values = [], ?string $order = null, array $custom = [])
+    {
+        if ([] !== $values) {
```

Comment:
are this `[] !== $values` checks really needed? `array_key_exists` should work well even if the array is empty

Reply:
I've just pushed a commit with this suggestion :man_dancing: